app/controllers/movies_controller.rb

```diff
@@ -1,29 +1,19 @@
 class MoviesController < ApplicationController
-	
-	def index
-		@movie_list = []
-
-		Movie.all.each do |movie|
-			showings = movie.showings
 
-			screen = {movie: movie, showings: showings}
-			@movie_list.push(screen)
+	def index
+		@movie_list = Movie.all.includes(:showings).map do |movie|
```

Comment:
Would this be something like having a scoped date where instead of grabbing all movies and showings I am only grabbing movies that have a showing between the dates that are displayed on the site?